operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/PrimaryToSecondaryIndex.java

```diff
@@ -0,0 +1,47 @@
+package io.javaoperatorsdk.operator.processing.event.source.informer;
+
+import java.util.*;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.javaoperatorsdk.operator.processing.event.ResourceID;
+import io.javaoperatorsdk.operator.processing.event.source.SecondaryToPrimaryMapper;
+
+public class PrimaryToSecondaryIndex<R extends HasMetadata> {
```

Comment:
No, it is also used by the user in case to define custom mapping. (We have default by owner reference, but that does not work for example outside of namespace. See also support in `KubernetesDependentResource`.

Reply:
I don't understand what you mean. This class is only used by `InformerEventSource` internally and since it's the class implementing the automated indexing of secondary resources, it shouldn't be exposed anyway.

Reply:
Sorry, I though this is the mapper. Yes this can be package private, will change it.

Reply:
fixed.